.github/workflows/build.yml

```diff
@@ -0,0 +1,47 @@
+name: Build
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  goreleaser:
+    name: Build
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with: { fetch-depth: 0 }
+
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with: { go-version-file: go.mod }
+
+      - name: Run GoReleaser
+        uses: goreleaser/goreleaser-action@v5
+        with:
+          version: latest
+          args: release --clean --snapshot --skip sign
+
+      - name: Archive artifacts for use in Docker build
+        uses: actions/upload-artifact@v4
```

Comment:
is this needed?

Reply:
Nope - good catch. I had it in initially when I copied the release workflow, and forgot to remove it when I simplified things.